### ISSUE TITLE
VVV2 already consists of dashboard folder. 

### ISSUE THREAD

**Issue by mukeshwani** [topdown/VVV-Dashboard#64]
Hi, I just finished setting up VVV2 and was attempting to clone this repo per the instructions however there is already a dashboard folder in my VVV/www/default/ folder. Should I override the folder or is there something else I'm missing.

**Comment by topdown** [comment]
I see what they did. I'll have to do some renaming and code changes.
Hang tight.

**Comment by tomjn** [comment]
Not at all, we moved the dashboard to a git repo, and it can now be overriden in `vvv-custom.yml` via:

```
dashboard:
  repo: ....
  branch: ...
```

**Comment by mamathemes** [comment]
I tried it that way, but it failed. I see the default VVV Dashboard. What is missing? 

dashboard:
  repo: https://github.com/topdown/VVV-Dashboard.git

**Comment by tomjn** [comment]
Hmm in which instructions are you being told to clone this repository? VVV
always clones a dashboard repository, you can change the repo it clones but
you should never need to do the deed yourself manually.

Whomever told you to manually clone the repo has provided bad instructions,
please indicate where these are

**Comment by mamathemes** [comment]
Please have a look at your comment here:

> Not at all, we moved the dashboard to a git repo, and it can now be overriden in vvv-custom.yml via:
> 
> dashboard:
>   repo: ....
>   branch: ...
> 
> It's available on the develop branch and lands in v2.2

**Comment by tomjn** [comment]
hmmm a miscommunication, delete the `dashboard` folder, modify `vvv-custom.yml` then reprovision, and it will checkout that repo/branch that was specified.

If that doesn't work I'll need to see the provisioner log
